boards/arm/gd32f407v_start/board.cmake

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022, TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=GD32F407VK" "--speed=4000")
+
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
this board should support openocd as well (just copy the support files from another f4xx board)

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/49085#discussion_r956536474